src/Profile/Profile.js

```diff
@@ -120,15 +121,13 @@ function Profile() {
           });
       } else {
         setUser(null);
-        navigate("/");
```

Comment:
Main reason of why the page was reloading twice, and wasn't displaying the banner for the new users.